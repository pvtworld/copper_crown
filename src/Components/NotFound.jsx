import React from 'react';

class Login extends React.Component{
    render() {
        return (
            <div>
                <h2>404 Page not found</h2>
                <p>This page doesn't exist or has been removed</p>
            </div>
        );
    }
}

export default Login;