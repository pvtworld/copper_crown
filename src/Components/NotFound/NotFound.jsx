import React from 'react';

 const NotFound = () => {
    return (
        <div>
            <h1>404</h1>
            <h4>The page you're looking for isn't here, go <a href="/">home</a> instead</h4>
        </div>
    );
};

export default NotFound;