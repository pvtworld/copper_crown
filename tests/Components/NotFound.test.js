import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import NotFound from '../../src/Components/NotFound/NotFound';

beforeAll(() => {
    
});

describe('Render test', ()=> {
    it('Renders without crashing', () => {
        const component = renderer.create(<NotFound />);

        expect(component).toMatchSnapshot();
    });
});