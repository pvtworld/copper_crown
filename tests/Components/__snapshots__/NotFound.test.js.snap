exports[`Render test Renders without crashing 1`] = `
<p>
  Sorry, whatever you\'re searching for is not here.
</p>
`;
